release_notes.txt

```diff
@@ -0,0 +1,104 @@
+===============================================================================
+                              Swift Navigation, Inc.
+                      Piksi Multi Firmware Release Notes
+
+Product Page:      swiftnav.com/piksi-multi
+Firmware Download: support.swiftnav.com
+===============================================================================
+
+
+-------------------------------------------------------------------------------
+Release 1.1.26                                                       2017-05-16
+-------------------------------------------------------------------------------
+FEATURES:
+* GPS L1CA/L2C RTK Operation
+* Low Latency (max 20 Hz) and Time Matched (max 5 Hz) RTK modes (updated feature)
+* RTK heading output (new feature)
+* Up to 200 Hz IMU raw data output
+* UART (two ports), USB (host and device), Ethernet interfaces
+* UART flow control (new feature)
+* Swift Binary (SBP), NMEA0183 and RTCMv3 protocols (updated feature)
+* Status LEDs (Power, Position, Link, Mode)
+* Active antenna supply control
+* External event trigger input (rising, falling or both edges)
+* 1 Pulse Per Second (PPS) output
+* Position Valid (PV) digital output
+* Surveyed position lat/lon/alt and all settings non-volatile storage
+* Simulator option
+* Onboard NTRIP client (new feature)
+* Onboard standalone data logger over usb flash media (new feature)
+
+CHANGES:
+* Improved general navigation performance and system stability.
+* RTCMv3 messages input added.
+* Standalone logging feature added.
+* SBP and NMEA heading output messages added.
+* Heading offset setting added.
+* Protocol mode added to Ethernet TCP/IP interfaces.
+* UART flow control added.
+* Accelerometer default range setting changed to 8g.
+* GPS time and velocity output added to the simulator.
+* UTC offset decoding added.
+* Observation data made compatible with post-processing tools.
+
+BUG FIXES:
+* [286] Not operational SBAS settings has been removed.
+* [278] USB device data buffer flushing added on new connection.
+* [233] NMEA messages time is provided only when fix is valid.
+* [235] NMEA GGA message rate is now configrable.
+* [269] USB interface NMEA mode added.
+* [234] NMEA GSA message reports used SVs.
+* [274] NMEA GSV message reports visible SVs.
+* [283] SBP messages MSG_UTC_TIME, MSG_DGPS_STATUS, MSG_AGE_CORRECTIONS have been implemented.
+
+KNOWN ISSUES:
+* [252] 1 PPS output is still generated after fix is lost.
+* [287] Incoming RTK corrections are echoed on the same port. It can create additional data traffic over the radio.
+* [230] Ethernet port LED does not show activity during communication.
+* [227] LED indicators may show incorrect status during reset.
+* [282] External event input has no debouncing. A sharp input edges are required to avoid multiple messages with

```

Comment:
There seems to be a new issue now though. 
https://github.com/swift-nav/piksi_v3_bug_tracking/issues/465